client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -87,7 +87,7 @@ const NotificationSubscriptions = createReactClass( {
 						onSubmit={ this.submitForm }
 					>
 						<FormSectionHeading>
-							{ this.props.translate( 'Subscriptions Delivery' ) }
+							{ this.props.translate( 'Subscriptions delivery' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **41** times:
`translate( 'Subscriptions Delivery' )` *ES Score: 12*